menas/src/main/scala/za/co/absa/enceladus/menas/controllers/SchemaController.scala

```diff
@@ -163,6 +181,15 @@ class SchemaController @Autowired()(
         throw notFound()
     }
   }
+
+  @GetMapping(path = Array("/availability"))
```

Comment:
Maybe `/features`.

Availability sounds like something related to 'high availability' and brings expectation of something being either 'available' or 'down'. But in this case the availability of a feature is related to the configuration of the service, not to its operation.

Reply:
I completely agree with this point, changed.